NGitLab.Mock/MergeRequestChangeCollection.cs

```diff
@@ -10,11 +11,25 @@ public MergeRequestChangeCollection(GitLabObject parent)
         }
 
         public Change Add(string diff)
```

Comment:
Should it be obsolete?

Reply:
Yes, would make things better in the end